app/api/api_extension.py

```diff
@@ -17,32 +26,33 @@ def ioslink():
     title="Mentorship System API",
     version="1.0",
     description="API documentation for the backend of Mentorship System. \n \n"
-    + "Mentorship System is an application that matches women in tech to mentor each other, on career development, "
+    + "Mentorship System is an application that matches women in tech to mentor each other, on career "
+    "development, "
     + "through 1:1 relations during a certain period of time. \n \n"
```

Comment:
Write it like this,
```suggestion
    "development, through 1:1 relations during a certain period of time. \n \n"
```

Reply:
Hi @devkapilbansal I think my Pycharm linter changed that. I used black formatter to reformat and apparently it missed that. Nevertheless, I'll change that. Thanks for the review :slightly_smiling_face: